tests/nfs-bug-5140/test.yaml

```diff
@@ -0,0 +1,12 @@
+pcap: nfsv2.pcap
```

Comment:
This is used to avoid duplicated pcaps in the SV repository, thus allowing one to use a pcap from another test. If the pcap is part of the same test directory, there's no need to add this.